### ISSUE TITLE
Accidental C style comments?

### ISSUE THREAD

**Issue by DRMacIver** [yyuu/pyenv#434]
Just tried to install pypy-2.6.1 and when it hit the "no binary distribution available" case it instead gave me a weird error: 

```
/home/david/.pyenv/plugins/python-build/share/python-build/pypy-2.6.1: line 32: //: Is a directory
```

The culprit seems to be https://github.com/yyuu/pyenv/commit/c9268bb0dc62ffacc8dbc0e40e3da07e041ccab1 where you are using C style // comments instead of shell style # comments. I don't know if there are shells that support that, but it's certainly not supported in bash. :-)

**Comment by yyuu** [comment]
Already fixed in 5ba585769b139198f9b2b31cf7b4f7ab24b67c9b

**Comment by joshfriend** [comment]
Thats my bad, sorry about that!
